src/app/api/auth/[auth0]/route.ts

```diff
@@ -0,0 +1,4 @@
+/* istanbul ignore file */
+import { auth0 } from "@/node-lib/auth/auth0";
+
+export const GET = auth0.handleAuth();
```

Comment:
These should really be using the edge runtime — but I had some trouble getting that running. We can look at that before release should we stick with Auth0. For now I didn't want to get into a fight with environment variables and the bespoke way we populate the `.env` in production.